contracts/optimistic-ethereum/libraries/codec/Lib_EIP155Tx.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+/* Library Imports */
+import { Lib_RLPReader } from "../rlp/Lib_RLPReader.sol";
+import { Lib_RLPWriter } from "../rlp/Lib_RLPWriter.sol";
+
+import { console } from "hardhat/console.sol";
+
+library Lib_EIP155Tx {
+    struct EIP155Tx {
```

Comment:
This could be reordered to fit in less words right?

Reply:
Hmmm. I looked into this. It appears that the tight packing only applies if we plan to store this struct. Since we're only using this in memory the order doesn't matter. Source here: https://docs.soliditylang.org/en/v0.8.2/internals/layout_in_memory.html#example-for-difference-in-struct-layout